inventory_service/src/item_actions.py

```diff
@@ -42,3 +42,4 @@ def update_item_container_action(self,data):
         return self.item_repo.update_item_container_repository(item_id,new_container)
     def delete_item_action(self,item_id):
         return self.item_repo.delete_item_respository(item_id)
+        
```

Comment:
Comment applies for all.
variable names can be improved
e.g.) at line 22
![image](https://user-images.githubusercontent.com/70574489/217152956-d14d7402-6a2d-4ea6-8414-ece83b538131.png)

print statements can be ignored
e.g) at line 24
![image](https://user-images.githubusercontent.com/70574489/217153027-9d3ef433-e32f-494d-a5f2-f63b95c16520.png)